src/error/error-handler.ts

```diff
@@ -17,6 +18,32 @@ export class ErrorHandler implements ExceptionFilter {
   }
 
   getResponseBody(error: unknown): { statusCode: HttpStatus; message: string } {
+    // forward class validation errors
+    if (error instanceof BadRequestException) {
+      const errorResponse = error.getResponse();
+
+      if (
+        typeof errorResponse === 'object' &&
+        errorResponse.hasOwnProperty('message') &&
+        errorResponse.hasOwnProperty('statusCode')
+      ) {
+        return {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          message: errorResponse?.message || 'VALIDATION_ERROR',
```

Comment:
1. I don't think marking `errorResponse` as `?` here is necessary. If statement already checked it's an object and has properties `message` and `statusCode`.
2. If we introduce some helper type for return type of `getResponseBody` function, like 

```typescript
type ErrorResponse = {
  message: string;
  statusCode: HttpStatus
}
```

we can change this code to `(errorResponse as ErrorResponse).message` and avoid `ts-ignore`s

Reply:
Ad. 1. This is odd but typescript was complaining about not checking the errorResponse even that I've checked if it is an object.

Ad. 2. Unfortunatelly the type is poorly typed as ```    getResponse(): string | object;``` :( 

I will add a typeguard instead of suppressing with ts-ignore.

Reply:
Applied type guard: 
https://github.com/copotrzebne-pl/copotrzebne-api/pull/42/files#diff-53191a1b9aeb6fcd0614b6e63f549ee41ea19cae93738d269d708005eb9e60a2R1-R13